src/main/webapp/projectReview/ProjectReview.js

```diff
@@ -528,6 +528,9 @@ const ProjectReview = hh(class ProjectReview extends Component {
       this.setState(prev => {
         prev.readOnly = true;
         prev.errorSubmit = false;
+        if (prev.formData.projectExtraProps.affiliations.value !== 'other') {
```

Comment:
Can affiliations be null? Please validate it

Reply:
Done!